### ISSUE TITLE
[Accessibility][Advisory][Image][Teaser]Linked image accessible name is read twice

### ISSUE THREAD

**Issue by LevelAccessA11y** [adobe/aem-core-wcm-components#1677]
**Steps to Reproduce**
1. Enable NVDA.
2. In a Chrome browser, navigate the Image component page with the TAB key to get to the linked glacier image.

**Current Behavior**
With NVDA running, the accessible name of the linked image is announced twice (e.g. in the Image component, the linked glacier image is announced "Snowy Mountain Glacier Graphic Snowy Mountain Glacier Link"). An unnecessary title attribute on the image duplicates the accessible name.

**Expected behavior/code**
The accessible name of the linked image should be announced only once to avoid repetition for screen reader users. 

**Environment**
Chrome 91, Windows 10, NVDA 2020.4 
AEM Core Components v. 2.17  

**Affected Users**
Screen reader users

**WCAG SC**
2.4.4 Link Purpose (In Context)

**Code snippet**
```
<a class="cmp-image__link" href="/content/core-components-examples/library/page-authoring/image.html" data-cmp-clickable="" data-cmp-hook-image="link">
      <img src="/content/core-components-examples/library/page-authoring/image/_jcr_content/root/responsivegrid/demo_1473529393/component/image.coreimg.jpeg/1550672497135/snowy-mountain-glacier.jpeg" class="cmp-image__image" itemprop="contentUrl" data-cmp-hook-image="image" alt="Snowy mountain glacier" title="Snowy mountain glacier">
</a>
```

**Possible Solution**
Consider removing the title attribute on the IMG.

**Comment by bpauli** [comment]
@adobe export issue to Jira project SITES

**Comment by paniciu** [comment]
@bpauli @LevelAccessA11y Tested this on my local environment, and there is already an option in the image component configuration to remove the title or change it to be different from the alt attribute. This can be set by going to the image component -> Configure -> Metadata -> Uncheck 'Display caption as pop-up' option.
![Image](https://user-images.githubusercontent.com/7657920/126788886-81b93a04-3a7b-4c61-88d0-766bb04701a2.png)

**Comment by bpauli** [comment]
@paniciu In this case we only need to configure this property for the component library.

**Comment by richardhand** [comment]
@paniciu, @bpauli, if not displaying the caption as popup, then it is rendered instead as a span. Seems like we're missing the ability to configure the component to display no caption at all (either as span or in the title attribute).

**Comment by gabrielwalt** [comment]
This can be fixed with the correct policy configuration on the teaser component: "Don't link the image".
However, in the new teaser version, we should make this the default, and not allow any option to change this in the policy.
